site/src/pages/components/file.mdx

```diff
@@ -95,12 +95,12 @@ button invokes the file selection prompt.
 
 ```html
 <host>
-  <div part="file-selector-button">
+  <button part="file-selector-button">
     <slot name="file-selector-button"></slot>
-  </div>
-  <div part="label">
+  </button>
+  <label part="label">
```

Comment:
this seems odd.  is this really its 'label', or is it more a description of the current state of the control.  e.g., no file chosen, x file(s) chosen?  If that latter, this probably shouldn't be a label element then.

Reply:
As per https://gist.github.com/lukewarlow/2eeb75ba6d3039cd10d850c19162976b It's a `<label>` in Firefox. A `<span>` in Blink and not represented in the Dom structure in WebKit. 

Having said that I'm happy to change it to `<span>` (or leave it be for now) if that's considered more appropriate.

Reply:
yeh, i don't really see the value in what firefox has there.  I'm actually rather [befuddled by chrome's implementation](https://github.com/w3c/html-aam/issues/421) - this actually reminded me i have bugs to file

Reply:
I've changed it to be a span to match the DOM structure of Chrome (and it's an inline element so it matches the actual rendering unlike a div)